manifests/params.pp

```diff
@@ -11,6 +11,27 @@
     fail('This module only works on Debian or derivatives like Ubuntu')
   }
 
+  $config_files = {
+    'conf'   => {
+      'path' => $conf_d,
+      'ext'  => '',
+    },
+    'pref'   => {
+      'path' => $preferences_d,
+      'ext'  => '',
+    },
+    'list'   => {
+      'path' => $sources_list_d,
+      'ext'  => '.list',
+    }
+  }
+
+  $file_defaults = {
+    'owner' => 0,
+    'group' => 0,
```

Comment:
Mmm no. I've just started to default to `0` instead of `root` because sometimes people rename the freaking root user. I can change it to `root`.